### ISSUE TITLE
Unit Type shared entries

### ISSUE THREAD

**Issue by capitaladot** [BSData/horus-heresy#462]
_From @Millicant on October 10, 2016 18:48_

Another issue that may or may not need solving: the various Rites of War that require/restrict certain unit types.  

As you mentioned @capitaladot the shared entries for things like Terminator Armour should make some of those easier, but I'm considering adding more for all units to allow implementation.  That way, a shared entry can easily have modifiers attached to it that enforce the requirements.  

This does not, however, properly hide/show entries based on the Rite of War chosen, but instead only informs you when you have illegally added a unit.  

Short list of potential unit types:
- Infantry (/Jump Infantry?)
- Tank
- Flyer
- Transport
- (Jet)Bike
- Immobile
- Deep Strike
- Skimmer 
- Slow and Purposeful
- Superheavy
- Others per various Rites of War

Does this seem like it would work?  Or too much work?  

_Copied from original issue: BSData/horus-heresy-bs2prep#39_

**Comment by capitaladot** [comment]
Awesome idea! Another valid set of required/counted descriptors are Tank (Ironwing), Deep Strike (Orbital Assault, Coils of the Hydra), Dreadnought (Fury of the Ancients) Transport (Ironwing, Coils of the Hydra) and Infiltrate (Coils of the Hydra).

**Comment by capitaladot** [comment]
_From @Millicant on October 10, 2016 18:58_

That's a good word for it: descriptors.  They aren't just unit types, they are descriptors that can be associated to certain conditions with the various Rites of War.

**Comment by capitaladot** [comment]
Maybe they're ultimately Rules?

**Comment by capitaladot** [comment]
_From @Millicant on October 10, 2016 19:11_

While that would be less obtrusive, they can't be conditionally required/restricted if they're only rules.  

I think they'll have to be Shared Entries.  I'll start going over the Rites of War for the various required "Descriptors" so we can compile a full list before we start adding anything.

Some can be handled smoothly without adding more entries, like Terminator Amour or Jump Packs with their existing shared entries.  The rest will have to be added as appropriate.
